src/test/java/LaunutsApiTest.java

```diff
@@ -84,19 +89,95 @@ public static void setUpTTL() throws IOException, JSONException {
         httpClient.close();
     }
 
+    @BeforeClass
+    public static void setUpLauJson() throws IOException, JSONException {
+        String jsonUrl = "http://localhost:8080/launuts/lau/de_09577125/json";
+
+        CloseableHttpClient httpClient = HttpClients.createDefault();
+        HttpGet httpJsonGet = new HttpGet(jsonUrl);
+        CloseableHttpResponse httpResponse = httpClient.execute(httpJsonGet);
+
+        HttpEntity entity = httpResponse.getEntity();
+        String content = EntityUtils.toString(entity);
+
+        JSONArray jsonArray = new JSONArray(content);
+
+        for (int i = 0; i < jsonArray.length(); i++) {
+            JSONObject jsonobject = jsonArray.getJSONObject(i);
+            lauId = jsonobject.getString("lauId");
+            lauName = jsonobject.getString("lauName");
+            lauCoordinates = jsonobject.getJSONArray("coordinates");
+        }
+
+        httpClient.close();
+    }
+
+    @BeforeClass
+    public static void setUpLauTTL() throws IOException, JSONException {
+
+        String urlTTL = "http://localhost:8080/launuts/lau/Dittenheim/ttl";
+
+        CloseableHttpClient httpClient = HttpClients.createDefault();
+        HttpGet httpTLlGet = new HttpGet(urlTTL);
+        CloseableHttpResponse httpResponseTTL = httpClient.execute(httpTLlGet);
+
+        StringWriter writer = new StringWriter();
+        IOUtils.copy(httpResponseTTL.getEntity().getContent(), writer, "UTF-8");
+        String fixedString = writer.toString();
+        Model defaultModel = ModelFactory.createDefaultModel();
+        defaultModel.read(new StringReader(fixedString),
+                urlTTL,
+                "TURTLE");
+
+        Resource resource = defaultModel.createResource("http://projekt-opal.de/launuts/lau/DE/09577122");
+
+        Statement statementlauCode = defaultModel.getProperty(resource, notation);
+        lauIdTTL = statementlauCode.getObject().toString();
+
+        Statement statementPreflabel = defaultModel.getProperty(resource, prefLabel);
+        lauNameTTL = statementPreflabel.getObject().toString();
+        httpClient.close();
+    }
+
+    @BeforeClass
+    public static void negativeTest() throws IOException, JSONException {
+
+        String urlTTL = "http://localhost:8080/launuts/lau/Dittenim/json";
+
+        CloseableHttpClient httpClient = HttpClients.createDefault();
+        HttpGet httpTLlGet = new HttpGet(urlTTL);
+        CloseableHttpResponse httpResponse = httpClient.execute(httpTLlGet);
+
+        HttpEntity entity = httpResponse.getEntity();
+        String content = EntityUtils.toString(entity);
+        JSONArray jsonArray = new JSONArray(content);
+
+        for (int i = 0; i < jsonArray.length(); i++) {
```

Comment:
Please skip the for loop, we need only one object. Instead, just get the first element as follows:

JSONObject jsonobject = jsonArray.getJSONObject(0);  

Same for all for loops....